modules/local/slr/main.nf

```diff
@@ -0,0 +1,41 @@
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process SLR {
+  tag "$meta"
+  label 'process_high'
+  
+  container "gitlab.lrz.de:5005/beckerlab/container-playground/slr:8878afbd"
+
+  publishDir "${params.out}",
+      mode: params.publish_dir_mode,
+      saveAs: { filename -> saveFiles(filename:filename,
+                                      options:params.options, 
+                                      publish_dir:"${task.process}".replace(':','/').toLowerCase(), 
+                                      publish_id:meta) }
+  input:
+      tuple val(meta), path(assembly), path(reads)
+  
+  
+  output:
+      tuple val(meta), path("${meta}_slr_scaffold.fa"), emit: scaffolds
+      tuple val(meta), path("${meta}_slr_unique.fa"),   emit: unique_contigs
+      tuple val(meta), path("${meta}_slr_ambigous.fa"), emit: abigous_contigs
+  
+  script:
+      def prefix = task.ext.prefix ?: "${meta}"
+  """
+  bwa index ${assembly}
```

Comment:
This looks like it would be a local module, or split between two given the SLR tool.
As the tools look common to a mulled container, you might be asked to use that so conda is also available to use.

Reply:
This module has been deprecated since the tool caused problems and two long-read scaffolders and reference based scaffolder seem to be sufficient..